terraform/modules/core/core.tf

```diff
@@ -506,14 +511,34 @@ resource "aws_appautoscaling_policy" "down" {
 
 /* metric used for auto scale */
 resource "aws_cloudwatch_metric_alarm" "service_cpu_high" {
-  alarm_name          = "${var.project_name}-${var.environment}_web_cpu_utilization_high"
+  alarm_name          = "${var.project_name}-${var.environment}-web-average-utilized-cpu-high"
   comparison_operator = "GreaterThanOrEqualToThreshold"
-  evaluation_periods  = "5"
+  evaluation_periods  = "2"
   metric_name         = "CPUUtilization"
   namespace           = "AWS/ECS"
   period              = "60"
   statistic           = "Average"
-  threshold           = "85"
+  threshold           = "30"
+
+  dimensions {
+    ClusterName = "${var.ecs_cluster_name}"
+    ServiceName = "${var.ecs_service_web_name}"
+  }
+
+  alarm_actions = ["${aws_appautoscaling_policy.up.arn}"]
+  ok_actions    = ["${aws_appautoscaling_policy.down.arn}"]
+}
+
+/* metric used for auto scale */
+resource "aws_cloudwatch_metric_alarm" "service_memory_high" {
```

Comment:
Memory is actually quite high (90%) on production while CPU remains low, this should also cause containers to be added